.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ["1.19", "1.18.5"]
+        go: ["1.19.1", "1.18.6"]
```

Comment:
Can't we use variables for all the Gitlab yamls so we do not need to update these in so many places?
We risk being inconsistent

I assume we can at least do it per file, right?

Reply:
Agree with that, @josvazg but it looks unrelated to the current issue..

Reply:
I agree with @agarcia-oss. Let's tackle that as a separate issue.

Reply:
Yes, separate issue makes sense.